clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -66,7 +72,35 @@ export default async function handler(
 
   // Check if a geolocation was provided via headers, query param, or obtainable via a geolocation URL;
   // if so, inject into the bundle, along with privacy experience
-  const geolocation = await lookupGeolocation(req);
+  const geolocation = await lookupGeolocation(req, environment.settings);
+
+  let experience;
```

Comment:
nit: this is really important (and powerful!) logic so a code comment is always a friend.
```suggestion
  // If a geolocation can be determined, "prefetch" the experience from the Fides API immediately.
  // This allows the bundle to be fully configured server-side, so that the Fides.js bundle can initialize instantly!
  let experience;
```